src/tests/EdFi.DataManagementService.Tests.E2E/Features/SubclassResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,37 @@
+Feature: Subclass resources validatio
+
+        Background:
+            Given this Local Education Agency
+                  | localEducationAgencyId | nameOfInstitution | localEducationAgencyCategoryDescriptor | categories                 |
+                  | 255901                 | Grand Bend ISD    | ["Independent"]                        | ["Local Education Agency"] |
+
+        @ignore
+        Scenario: Ensure client can't get create a school with the same ID as another Subclass
+             When a POST request is made to "/ed-fi/schools" with
+                  """
+                  {
+                      "schoolId": 255901,
+                      "nameOfInstitution": "School Test",
+                      "educationOrganizationCategories": [
+                          {
+                              "educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#Educator Preparation Provider"
+                          }
+                      ],
+                      "gradeLevels": [
+                          {
+                              "gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Postsecondary"
+                          }
+                      ]
+                  }
+                  """
+             Then it should respond with 409
+              And response body is
```

Comment:
This should be `"And the response body is"`

Reply:
Changed